cpp/autosar/src/rules/A0-1-3/UnusedLocalFunction.ql

```diff
@@ -75,7 +113,22 @@ where
     // There exists an instantiation which is called
     functionFromInstantiatedTemplate.isConstructedFrom(functionFromUninstantiatedTemplate) and
     functionFromInstantiatedTemplate = getTarget(_)
-  ) and
+  )
+  and
+  // A function is defined as "used" if any one of the following holds true:
+  // - It's an explicitly deleted functions e.g. =delete
+  // - It's annotated as "[[maybe_unused]]"
+  // - It's part of an overloaded set and any one of the overloaded instance
+  //   is called.
+  // - It's an operand of an expression in an unevaluated context.
+  (
```

Comment:
Minor, but these brackets are superfluous and I would suggest removing them.

Reply:
Removed in b150aed635c785aae01548540dde8ed9ac9152e3